src/config.py

```diff
@@ -489,7 +493,8 @@ def client_properties(self) -> List[str]:
         client_properties = [
             f'sasl.jaas.config=org.apache.kafka.common.security.scram.ScramLoginModule required username="{username}" password="{password}";',
             "sasl.mechanism=SCRAM-SHA-512",
-            f"security.protocol={self.security_protocol}",  # FIXME: will need changing once multiple listener auth schemes
+            f"security.protocol={self.security_protocol}",
+            # FIXME: will need changing once multiple listener auth schemes
```

Comment:
nit: since it's not inline with the value, maybe specify the key to not confuse between security protocol and bootstrap servers.
```suggestion
            # FIXME: security.protocol will need changing once multiple listener auth schemes
```